helm/istio/charts/kiali/templates/kiali-cr.yaml

```diff
@@ -36,22 +36,28 @@ spec:
     grafana:
       auth:
         ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
-        type: "bearer"
-        use_kiali_token: true
+        type: "basic"
+        use_kiali_token: false
+        username: "internal"
+        password: ""
```

Comment:
There's no way to specify a secret for this?

Reply:
As discussed on IRC: No, not at the moment.